### DISCUSSION TITLE
BB plotter with instant estimates of supported farm size

### DISCUSSION THREAD

**Discussion by brianwfreeman** [Chia-Network/bladebit#427] [Ideas]
One feature of NoSSD that I hope BB borrows is that when you start the plotting process it tells you how many plots the GPU can handle for farming. I know we have the more robust --simulate switch, but NoSSD just makes it very simple, which is a good thing.

**Comment by hajes** [comment]
if you do not know what your HW can handle, how can you start plotting? it makes no sense ;-) simulate, test farm, replot...no need for fancy/useless stuff ;-)
